dotCMS/src/main/java/com/dotcms/rest/config/DotRestApplication.java

```diff
@@ -170,6 +172,7 @@
 			.add(VersionableResource.class)
 			.add(PermissionResource.class)
 			.add(ContentResource.class)
+			.add(CacheResource.class)
```

Comment:
Shouldn't `RedisResource` be added here?

Reply:
nope, b/c it was just for own testing